compute_endpoint/packaging/debian/changelog.in.in

```diff
@@ -0,0 +1,8 @@
+globus-compute-endpoint (@VERSION@.@distro@) @distro@; urgency=low
```

Comment:
This is a packaging detail that is largely irrelevant to us since we will host these files in our own, non-Debian non-Canonical repository.

However, I believe the answer is "vice-versa."  Please see [the relevant Debian Packaging Document section](https://www.debian.org/doc/manuals/developers-reference/best-pkging-practices.html#selecting-the-upload-urgency) for me details.  In particular, I interpret this sentence:

> For example, you might select **urgency=low** if the changes since the last upload are large and might be disruptive in unanticipated ways.

for our initial push as "it's a big change, so urgency low."

Our next update would likely be `medium`, however.